lesson02/assignment/index.html

```diff
@@ -1,15 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 02</title>
-    </head>
-    <body>
-        <main class="main">
-            <h1>title</h1>
-        </main>
-    </body>
+  <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+      <link rel="stylesheet" href="css/style.css" />
+      <title>Assignment 02</title>
+  </head>
+  <body>
+    <!-- Setting up two content sections for page and flexbox -->
+    <div class="main">
+      <div class="box1">
```

Comment:
Consider more meaningful classnames, like "intro", "bio", or "details" rather than "box1" and "box2", and "profile-card" or similar rather than "main". This will make maintaining the CSS for it easier in the future, as it'll be immediately clear what elements will be impacted when you're making changes in the CSS file.